src/mavedb/routers/taxonomies.py

```diff
@@ -0,0 +1,131 @@
+from typing import Any, List
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy import func, or_
+from sqlalchemy.orm import Session
+
+from mavedb import deps
+from mavedb.models.taxonomy import Taxonomy
+from mavedb.view_models import taxonomy
+from mavedb.view_models.search import TextSearch
+
+router = APIRouter(
+    prefix='/api/v1/taxonomies', tags=['taxonomies'], responses={404: {'description': 'Not found'}}
+)
+
+@router.get("/concatenateTaxonomyList", status_code=200, response_model=List[str], responses={404: {}})
```

Comment:
why do we need this API call? why is it called `concatenateTaxonomyList` specifically? how is this intended to be used?

Reply:
I shall delete this function. It was originally planned for used in dropdown menu, but later I noticed that I could fulfil it in UI side directly.